checks/checks.yaml

```diff
@@ -51,6 +51,14 @@ checks:
       - >-
         Signup for automatic dependency updates with dependabot or renovatebot and place the config
         file in the locations that are recommended by these tools.
+  Binary-Artifacts:
+    description: >-
+      This check tries to determine if a project has binary artifacts in the source repository.
+      The files with following extensions are checked ".o", ".so", ".ko", ".elf", ".exe", ".dll", ".deb", 
+      ".rpm", ".jar", ".pyc". These binaries could be compromised artifacts.
```

Comment:
IMO, we shouldn't mention the extensions/binaries we look for. This can easily change in the future. Also, this is tied to the implementation detail (looking for extensions vs. looking at file content) which can also change. Instead, let's just mention the high-level issue here (as @laurentsimon has suggested).

Reply:
+1, lets not put this in docs.